index.html

```diff
@@ -318,6 +318,12 @@ <h3>
                 <p>
                     The <dfn>pagePath</dfn> attribute contains the page path for current MiniApp.
                 </p>
+                
+                <aside class="note">
+                    <p>
+                        By default, the pagePath uses the value defined by <a href="https://w3c.github.io/miniapp-manifest/#pages-member">the first item in the MiniApp manifest's pages member</a>. If the value is not set by <a href="https://w3c.github.io/miniapp-manifest/#pages-member"> the MiniApp manifest's pages member</a>, the pagePath uses the path of current MiniApp's first page.
```

Comment:
Thanks @QingAn. Please consider changing the URL -> https://www.w3.org/TR/miniapp-manifest/#pages-member

Reply:
Thanks @espinr. The URL has been updated.